src/language-js/print/call-arguments.js

```diff
@@ -46,8 +46,17 @@ function printCallArguments(path, options, print) {
   }
 
   // useEffect(() => { ... }, [foo, bar, baz])
+  // useImperativeHandle(ref, () => { ... }, [foo, bar, baz])
   if (isReactHookCallWithDepsArray(args)) {
-    return ["(", print(["arguments", 0]), ", ", print(["arguments", 1]), ")"];
+    const parts = ["("];
+    for (let i = 0; i < args.length; i++) {
+      parts.push(print(["arguments", i]));
+      if (i !== args.length - 1) {
+        parts.push(", ");
+      }
+    }
+    parts.push(")");
+    return parts;
```

Comment:
This part can simplify to `join(..., path.map(...))`, but better to use `iterateCallArgumentsPath(...)`.

Reply:
```diff
@@ -49,12 +49,12 @@ function printCallArguments(path, options, print) {
   // useImperativeHandle(ref, () => { ... }, [foo, bar, baz])
   if (isReactHookCallWithDepsArray(args)) {
     const parts = ["("];
-    for (let i = 0; i < args.length; i++) {
-      parts.push(print(["arguments", i]));
-      if (i !== args.length - 1) {
+    iterateCallArgumentsPath(path, (arg, index) => {
+      parts.push(print(arg));
+      if (index !== args.length - 1) {
         parts.push(", ");
       }
-    }
+    });
     parts.push(")");
     return parts;
   }
~
``` 
This is how `iterateCallArgumentsPath` is used right?